Dockerfile

```diff
@@ -57,12 +55,9 @@ FROM ubuntu:focal
 
 LABEL maintainer="Nolan Brubaker <brubakern@vmware.com>"
 
-ARG BIN
```

Comment:
Do we need to update any invocations to the Dockerfile that used this argument, or were there none?

Reply:
I didn't see this arg being used anywhere for this stage in the Dockerfile. This change is particularly for running the container.